lib/right_chimp/Chimp.rb

```diff
@@ -1138,6 +1138,7 @@ def chimpd_wait_until_done
               all = ChimpDaemonClient.retrieve_group_info(@chimpd_host, @chimpd_port, @group, :all)
             rescue RestClient::ResourceNotFound
               sleep 5
+              $stdout.print "\nStill waiting for group #{@group}..."
```

Comment:
Can we actually change this to something along the lines of: WARN: Group "#{@group}" does not exist ? I tihnk that is a little bit more specific on what the issue really is.